src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/forslag/ForslagService.kt

```diff
@@ -0,0 +1,14 @@
+package no.nav.amt.deltaker.bff.deltaker.forslag
+
+import no.nav.amt.lib.models.arrangor.melding.Forslag
+import java.util.UUID
+
+class ForslagService(
```

Comment:
Hensikten med denne servicen er at den skal kunne gjøre mer senere :) F.eks. tenkte jeg den kan ha ansvar for å publisere avviste forslag til kafka-topicen.